### PR TITLE
🆕🎨 Design updates [consolidating branch]

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @fairlighteth , Swap button is not clickable in the PR: 
image

File: src/cow-react/modules/application/pure/Widget/index.tsx [hunk 1]
```diff
@@ -3,20 +3,14 @@ import { BodyWrapper as BodyWrapperMod } from 'pages/AppBody'
...
+  border-radius: 16px;
+  box-shadow: ${({ theme }) => theme.boxShadow1};
+  padding: 10px;
+  max-width: ${({ theme }) => theme.appBody.maxWidth.trade};
 
-  ${({ theme }) => theme.mediaWidth.upToSmall`
-    border: ${({ theme }) => theme.appBody.borderMobile};
-    box-shadow: ${({ theme }) => theme.appBody.boxShadowMobile};
-  `};
+  /* ${({ theme }) => theme.mediaWidth.upToSmall`
```
Comment (anxolin) [inline src/cow-react/modules/application/pure/Widget/index.tsx:12]: dead code

File: src/cow-react/modules/limitOrders/pure/Orders/styled.ts [hunk 2]
```diff
@@ -2,6 +2,8 @@ import styled from 'styled-components/macro'
 import { Widget } from '../Widget'
 
 export const Orders = styled(Widget)`
-  margin-top: 1.2rem;
+  /* TODO: not sure about this change! commented out from latest develop, the one is line 7 from design update branch */
```
Comment (anxolin) [inline src/cow-react/modules/limitOrders/pure/Orders/styled.ts:5]: who can double check this?

File: src/custom/theme/baseTheme.tsx [hunk 3]
```diff
@@ -23,19 +24,30 @@ export function colors(darkMode: boolean): Colors {
...
-    // ****** base ******
-    white: darkMode ? '#c5daef' : '#ffffff',
-    black: darkMode ? '#021E34' : '#000000',
+    // CoW Swap V2 colors ======================
+    white: darkMode ? '#CAE9FF' : '#ffffff',
+    black: '#07162D',
+    blueDark1: '#07162D',
+    blueDark2: '#052B65',
+    blueLight1: '#CAE9FF',
+    grey1: darkMode ? '#07162D' : '#ECF1F8',
```
Comment (anxolin) [inline src/custom/theme/baseTheme.tsx:33]: Is it a good practice to add "colors" as names in the vars? shouldn't be more semantic instead?

File: src/custom/theme/cowSwapTheme.tsx [hunk 4]
```diff
@@ -37,29 +36,36 @@ const mediaWidthTemplates: { [width in keyof typeof MEDIA_WIDTHS]: typeof css }
...
-    return theme(darkMode, shouldBlurBackground)
-  }, [darkMode, location.pathname, tradeTypeInfo])
+    return theme(
+      darkMode
+      // shouldBlurBackground
+    )
+  }, [
+    darkMode,
+    // location.pathname,
+    // tradeTypeInfo,
```
Comment (anxolin) [inline src/custom/theme/cowSwapTheme.tsx:67]: can you kill this dead code if we don't need it anymore?

File: src/components/CurrencyInputPanel/index.tsx [hunk 5]
```diff
@@ -4,7 +4,7 @@ import { Pair } from '@uniswap/v2-sdk'
 import { useWeb3React } from '@web3-react/core'
 import { AutoColumn } from 'components/Column'
 import { LoadingOpacityContainer, loadingOpacityMixin } from 'components/Loader/styled'
-import { darken } from 'polished'
+import { darken, lighten } from 'polished'
```
Comment (anxolin) [inline src/components/CurrencyInputPanel/index.tsx:7]: We should not mod this file. This is an original

**Review by anxolin — APPROVED** [review]
Soft approve, with comments. The most important thing is to not modify that original. All the others are nitpicks.

File: src/custom/theme/baseTheme.tsx (hunk 3 above)
Comment (fairlighteth) [reply src/custom/theme/baseTheme.tsx:33]: Yes makes sense, though that makes some of those names really long.

**Comment by elena-zh** [comment]
Hey @fairlighteth , as I promised, I will create separate issues related to the bug I find while testing this PR. I will separate issues per page/area. So I have already reported #1404  bug with issues related to Tokens/Account page. I will add as comments to this PR further issues I will report.

**Comment by elena-zh** [comment]
Issues related to token lists: #1405

**Comment by elena-zh** [comment]
Issues related to app general layout and content pages:  #1406

**Comment by elena-zh** [comment]
Issues related to Network settings #1407

**Comment by elena-zh** [comment]
Issues related to Swap container and orders #1408

Important issue here is that I can't cancel a pending order there (11 case)

**Comment by shoom3301** [comment]
@fairlighteth Do you need help with fixing tests?

**Pull request merged**